File: parte001/ex012_lados_triangulo.js
// Dados  los  tres  lados  de  un  triángulo,  haga  un  algoritmo  que  encuentre:  perímetro, 
// semiperímetro y el área del triángulo.

const lado1 = 5;
const lado2 = 6;
const lado3 = 7;

const perimetro = lado1 + lado2 + lado3;
const semiperimetro = perimetro / 2;
const area = Math.sqrt(semiperimetro * (semiperimetro - lado1) * (semiperimetro - lado2) * (semiperimetro - lado3));

console.log("El perímetro del triángulo es: " + perimetro);
console.log("El semiperímetro del triángulo es: " + semiperimetro);
console.log("El área del triángulo es: " + area);

File: parte001/ex012_lados_triangulo.cs
// Dados  los  tres  lados  de  un  triángulo,  haga  un  algoritmo  que  encuentre:  perímetro, 
// semiperímetro y el área del triángulo.

using System;

namespace Parte001
{
    class Program
    {
        static void Main(string[] args)
        {
            double lado1, lado2, lado3, perimetro, semiperimetro, area;

            Console.Write("Ingrese el valor del lado 1 del triángulo: ");
            lado1 = Convert.ToDouble(Console.ReadLine());

            Console.Write("Ingrese el valor del lado 2 del triángulo: ");
            lado2 = Convert.ToDouble(Console.ReadLine());

            Console.Write("Ingrese el valor del lado 3 del triángulo: ");
            lado3 = Convert.ToDouble(Console.ReadLine());

            perimetro = lado1 + lado2 + lado3;
            semiperimetro = perimetro / 2;
            area = Math.Sqrt(semiperimetro * (semiperimetro - lado1) * (semiperimetro - lado2) * (semiperimetro - lado3));

            Console.WriteLine("El perímetro del triángulo es: " + perimetro);
            Console.WriteLine("El semiperímetro del triángulo es: " + semiperimetro);
            Console.WriteLine("El área del triángulo es: " + area);

            Console.ReadKey();
        }
    }
}
